WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -870,8 +840,13 @@ - (void)cleanupAfterRefresh
 
 - (void)tableViewHandlerWillRefreshTableViewPreservingOffset:(WPTableViewHandler *)tableViewHandler
 {
-    [UIView performWithoutAnimation:^{
-        [self cleanupAfterRefresh];
+    [[self managedObjectContext] performBlockAndWait:^{
```

Comment:
Ah! sorry I missed this. I removed the performBlockAndWait from the new list but I missed removing it here.  Thanks for the ping!